Cargo.toml

```diff
@@ -10,10 +10,12 @@ repository = "https://github.com/cloudhead/nonempty"
 [dependencies]
 serde = { features = ["serde_derive"], optional = true, version = "1" }
 arbitrary = { features = ["derive"], optional = true, version = "1" }
+bincode = { optional = true, version = "2.0.0-rc.3" }
```

Comment:
Yeh, I understand that but this RC version has been around for more than a year now and is downloaded 2-3k times on a bad day and 10-40k times on a good/very good day (it's been downloaded more than 3M times already). Also `bincode` has had 2.0 RC versions for the past 3 years so I'd assume it's nearing completion, especially since the latest `1.x` version was released more than 3 years ago.

Finally, just to be clear, `bincode` `1.3` uses `serde` so `nonempty` is already compatible with it.

Reply:
Ah ok, sorry I didn't look past the version number :) Then that's fine :ok_hand:

Reply:
No worries 😁